metricq/source.py

```diff
@@ -89,31 +89,26 @@ async def task(self):
 
     """
 
-    def __init__(self, *args, **kwargs):
-        super().__init__(*args, **kwargs)
-        self.metrics = dict()
-        self._chunk_size: int = 1
+    chunk_size: Optional[int] = cast(Optional[int], ChunkSize())
+    """Number of :term:`data points<Data Point>` collected into a chunk before being sent.
```

Comment:
This whole description might not be clear because the chunk size is counted **per** metric. This might leave people with the expression that chunking is done for all metrics combined.

Reply:
Does bf31ab8099ae1da5cffb6b2414f144d8ece1be0e make this clear?